src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: '48 Laws of Power',
+      author: 'Robert Greene',
+      category: 'Personal Development',
+    },
+    {
+      id: 2,
+      title: 'Personal Development',
+      author: 'Jim Rohn',
+      category: 'Personal Development',
+    },
+  ],
+};
```

Comment:
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but I'd kindly suggest you carefully read the project requirements before making changes to your code base as it's specifically stated that the initial state of the categories should be an empty array as shown below 👇, not an array of books.
```jsx
initialState: {
    categories: [],
  },
```